exonum/src/node/state.rs

```diff
@@ -816,10 +816,7 @@ impl State {
                 for hash in msg.transactions() {
                     if transactions.get(hash).is_some() {
                         if !transaction_pool.contains(hash) {
-                            bail!(
-                                "Received propose with already\
-                                 committed transaction"
-                            )
+                            warn!("Received propose with already committed transaction");
```

Comment:
Node should never send propose with transaction that committed before. This is byzantine behaviour. Tx order in `Block` is significant, we can't ignore part of transactions in `Propose`.

Reply:
That's exactly what I've asked today on the meeting about (probably in wrong words).
If this behavior is not acceptable, then I guess there is nothing to be fixed in this ticket.
Fix of ECR-1251 fixes this bug too:
Committed transaction can't get into pool -> committed transaction can't get into proposal.

So this PR and ticket may be closed, I guess.